sshmux.go

```diff
@@ -359,6 +359,50 @@ func sendLogAndClose(logMessage *LogMessage, session *ssh.PipeSession, logCh cha
 	logCh <- *logMessage
 }
 
+func sshmuxListenAddr(address string, sshConfig *ssh.ServerConfig, proxy bool, proxyMux bool) {
```

Comment:
Yet another ugly workaround… I wish there were type-safe `Enum`s built-in in Go.